### DISCUSSION TITLE
Czy istnieje możliwość zablokowania użytkownikom rejestracji w organizacji?

### DISCUSSION THREAD

**Discussion by Grelo4ka** [AzureCommunityPL/group#101] [Pomoc] [answered]
Moja organizacja chciałaby usunąć użytkownikom możliwość rejestracji w organizacji, tzn. przy użyciu adresów służbowych (z domeną naszej organizacji). Czy istnieje sposób na wprowadzenie takiego organiczenia? Nowi użytkownicy byliby dodawani wyłącznie przez administratora z poziomu panelu administracyjnego.

**Comment by kamil-mrzyglod** [comment]
Ja to ciemna masa jestem - możesz napisać nieco więcej o co więcej chodzi? AAD? O jakim procesie rejestracji mówisz?

**Reply by Grelo4ka** [reply]
Użytkownicy mogą obecnie założyć konto w Microsoft Teams przy użyciu szkolnego adresu mailowego na stronie https://www.microsoft.com/pl-pl/education/products/office . Konta zakładalibyśmy na stronie https://admin.microsoft.com/

**Reply by kamil-mrzyglod** [reply]
To nie wiem czy nie lepszym miejscem na tego typu pytanie będzie https://www.facebook.com/groups/m365ugpl 😉

**Reply by smereczynski** [reply]
Pytanie nie dotyczy Azure. Usuń proszę wątek @Grelo4ka

**Reply by Grelo4ka** [reply]
> Pytanie nie dotyczy Azure. Usuń proszę wątek @Grelo4ka

Niestety nie jestem w stanie usunąć wątku, mogę go tylko edytować

**Reply by kaluzaaa** [reply]
@smereczynski tym razem się myślisz, to właściwość Azure AD tylko bardzo bardzo mało znana :-)

**Reply by smereczynski** [reply]
@kaluzaaa ale pytanie dotyczy Teams.

**Reply by smereczynski** [reply]
@Grelo4ka dobrze wiedzieć :)

**Comment by kaluzaaa** [comment, marked as answer]
@Grelo4ka dokumentacja -> https://docs.microsoft.com/en-us/azure/active-directory/users-groups-roles/directory-self-service-signup
